setup.py

```diff
@@ -131,7 +131,7 @@
     "multiprocess",
     # to save datasets locally or on any filesystem
     # minimum 2023.1.0 to support protocol=kwargs in fsspec's `open`, `get_fs_token_paths`, etc.: see https://github.com/fsspec/filesystem_spec/pull/1143
-    "fsspec[http]>=2023.1.0,<=2023.10.0",
+    "fsspec[http]>=2023.1.0",
```

Comment:
Was unsure if the preference here is to remove the upper bound or to increase the range to the current fsspec release (`<=2024.2.0`)